src/compas_timber/ghpython/components/CT_Joint_Rule_Topology/metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Topological Joint Rules",
+    "nickname": "JointsByTopo",
+    "category": "COMPAS Timber",
+    "subcategory": "Joints",
+    "description": "makes rules to apply joint types for each topology type.",
+    "exposure": 2,
+    "ghpython": {
+        "isAdvancedMode": true,
+        "iconDisplay": 0,
+        "inputParameters": [
+            {
+                "name": "L",
+                "description": "Joint type to be applied to L topology. Default type is L-Miter. If L-Butt, the beam with a larger cross section area will be used as crossing beam.",
```

Comment:
expected input is coming from `JoinOptions` components, right? maybe mention this here.